build-system/dependencies.gradle

```diff
@@ -70,4 +72,8 @@ ext {
             powermockXstream    : "org.powermock:powermock-classloading-xstream:${powermockVersion}",
             espresso            : "com.android.support.test.espresso:espresso-core:${espressoVersion}",
     ]
+
+    localizationLibraries = [
+            localization: "com.akexorcist:localization:${localizationVersion}"
```

Comment:
Have you tried this lib? Or at least confirming that the issue isn't happening on the SDK itself or affecting the host app. I'm seeing some open issue may become critical if affecting our merchant's app :
- https://github.com/akexorcist/Localization/issues/85
- https://github.com/akexorcist/Localization/issues/75
- https://github.com/akexorcist/Localization/issues/80

Reply:
yes we have tested it. and also the merchant have tested that our sdk using that lib. and if I use release build it's still running well. but let re-test by QA.

Reply:
@farhanpahlevi @askur87 Feel free to ask for help when you have any problem from my library in your SDK.

Reply:
Ok thanks @akexorcist. will let you know if we facing some issue with your library.